flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/SinkV2ITCase.java

```diff
@@ -111,12 +114,13 @@ public void writerAndCommitterExecuteInStreamingMode() throws Exception {
         final Source<Integer, ?, ?> source = createStreamingSource();
 
         env.fromSource(source, WatermarkStrategy.noWatermarks(), "source")
+                .rebalance()
```

Comment:
Why `rebalance()` is important here? (and if it is, can you add the answer as comment?)

ditto other places